### PR TITLE
Update "CSVDiagnosticsOutput" Docstring

### PR DESCRIPTION
**Pull request by GarethTMorgan**
Addresses the edit request from #29

### CODE REVIEW

**Comment by arichar6** [comment]
Can you change this PR to pull from your fork into the dev-documentation branch?

**Comment by GarethTMorgan** [comment]
Just changed that, thanks for the reminder @arichar6

File: turbopy/diagnostics.py [hunk 1]
```diff
@@ -20,8 +20,9 @@ class CSVDiagnosticOutput:
     ----------
     filename : str
        File name for CSV data file.
-    diagnostic_size : int
-       Size of data set to be written to CSV file.
+    diagnostic_size : (int, int)
+       Size of data set to be written to CSV file. First value is the number of
+       time points. Second value is number of spacial points.
```
Comment (arichar6) [inline turbopy/diagnostics.py:25]:
```suggestion
       time points. Second value is number of spatial points.
```

Comment (arichar6) [reply turbopy/diagnostics.py:25]: While both spellings are acceptable, _spatial_ is the more common spelling.

Comment (GarethTMorgan) [reply turbopy/diagnostics.py]: fixed the spelling error in the latest update.

**Pull request merged**